lib/routes/admin/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+var express = require('express');
+
+module.exports = function buildAdminRouter(mediator) {
+  var router = express.Router();
+
+  // TODO: Remove this endpoint when not needed
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
can you please add `NODE_ENV` to gruntfile, otherwise it's just another gotcha we need to remember.